crates/iota/src/iota_commands.rs

```diff
@@ -1216,23 +1223,18 @@ async fn prompt_if_no_config(
                         Err(e) => return Err(anyhow!("{e}")),
                     }
                 };
-                let (new_address, phrase, scheme) =
-                    keystore.generate_and_add_new_key(key_scheme, None, None, None)?;
-                let alias = keystore.get_alias_by_address(&new_address)?;
+                let (new_address, phrase, scheme) = config
+                    .keystore_mut()
+                    .generate_and_add_new_key(key_scheme, None, None, None)?;
+                let alias = config.keystore().get_alias_by_address(&new_address)?;
                 println!(
                     "Generated new keypair and alias for address with scheme {:?} [{alias}: {new_address}]",
                     scheme.to_string()
                 );
                 println!("Secret Recovery Phrase : [{phrase}]");
-                new_address
-            };
-            let alias = env.alias().clone();
-            IotaClientConfig::new(keystore)
-                .with_envs([env])
-                .with_active_address(active_address)
-                .with_active_env(alias)
```

Comment:
Removed?

Reply:
I meant just `with_active_env`

Reply:
It's done above instead

Reply:
Where?

Reply:
Oh, I forgot. I removed it because the first env is the default anyway, so it does not matter.